src/js/main.js

```diff
@@ -387,7 +389,7 @@ Site.Map = {
 
     // Add a marker for each location
     if (WP.locations.length) {
-      for(var i = 0; i < WP.locations.length; i++) {
+      $(WP.locations).each(function(i) {
```

Comment:
You will get `(index, item)` in this callback if you want. So you don't need the `WP.locations[i]` part. Also I think `i` is a valid name for an iterator but when using the callback like this `index` is nicer. But that is just style